saleor/graphql/order/mutations/orders.py

```diff
@@ -339,11 +347,9 @@ def perform_mutation(cls, _root, info, **data):
         clean_order_update_shipping(order, method)
 
         order.shipping_method = method
-        shipping_price = info.context.plugins.calculate_order_shipping(order)
+        shipping_price = calculations.order_shipping(order, manager)
```

Comment:
AFAIR we wanted to delay calculations of new prices, I think this should be also done for shipping. This is why we have a flag expiration for prices. The calculation should be done, when storefront/dashboard calls an expected resolver that needs to have calculated prices.